source/mir/cv/core/memory.d

```diff
@@ -0,0 +1,152 @@
+/+
+    Module contains memory handing routines used throughout the library.
++/
+module mir.cv.core.memory;
```

Comment:
Could you please create a repo `mir-rt` and copy [glas/internal/memory](https://github.com/libmir/mir-glas/blob/master/source/glas/internal/memory.d) here to be reused both in mir-cv and mir-glas? It is already contains required pragmas.